events/yaml-sync.js

```diff
@@ -16,6 +17,22 @@ exports.getRegion = function (region) {
   return reg
 }
 
+exports.isSoT = function(region, city) {
+  for (var reg in store.regions) {
+    if (store.regions[reg].region == region) {
+      var meetups = store.regions[reg].meetups
+      for (var i in meetups) {
+        if (meetups[i].city == city) {
```

Comment:
I think we should find a better way to match this. What happens if there are 2+ events from the same region and city?